crrExistingObjects.js

```diff
@@ -62,14 +63,50 @@ function _markObjectPending(bucket, key, versionId, repConfig, cb) {
         }, next),
         // update replication info and put back object blob
         (mdRes, next) => {
-            const objMD = JSON.parse(mdRes.Body);
+            objMD = JSON.parse(mdRes.Body);
+            if (objMD.replicationInfo && objMD.replicationInfo.status !== '') {
```

Comment:
Maybe it's overkill to create a `ObjectMD` instance to check the values (e.g. using the methods `getReplicationInfo` and `getReplicationStatus`), but it would have the advantage of referring to a source of truth for the metadata schema. However, it could require extra work in Arsenal to handle backward compatibility with older object metadata versions. Just thinking out loud here. 😸

Reply:
Actually, sounds like a good idea, but when I tried using an arsenal dependency to use some versioning helper I ran into some weird compatibility error between the node.js version used by "diskusage" module and this s3utils script, where I assume "diskusage" is pulled by arsenal in a dependency chain, so I will keep as is. It would be good to understand if some time we have to pull this dependency for another reason.